src/content/plugins/chunks-webpack-plugin.md

```diff
@@ -0,0 +1,322 @@
+
+```js
+new ChunksWebpackPlugin({
+  fileExtension: '.html'
+});
+```
+
+### `templateStyle`
+
+`string = '<link rel="stylesheet" href="{{chunk}}" />'`
+
+Tells the plugin whether to personalize the default template for the HTML `<style>` tags. For example, add additional attributes, a CDN prefix or something else.
+
+```js
+new ChunksWebpackPlugin({
+  templateStyle: '<link rel="stylesheet" href="{{chunk}}" />'
+});
+```
+
+T> Keep the `{{chunk}}` placeholder, it is automatically replaced by the concatenation of the webpack public path and the chunk filename.
+
+### `templateScript`
+
+`string = '<script src="{{chunk}}"></script>'`
+
+Tells the plugin whether to personalize the default template for the HTML `<script>` tags. For example, add additional attributes, a CDN prefix or something else.
+
+```js
+new ChunksWebpackPlugin({
+  templateScript: '<script src="{{chunk}}"></script>'
+});
+```
+
+T> Keep the `{{chunk}}` placeholder, it is automatically replaced by the concatenation of the webpack public path and the chunk filename.
+
+### `customFormatTags`
+
+`false || function (chunksSorted, files)`
+
+Tells the plugin whether to personalize the default behavior for generate your own templates. The function is called for each entry points. Custom behavior can also be add for a specific entry point if necessary.
+
+```js
+new ChunksWebpackPlugin({
+  customFormatTags: (chunksSorted, Entrypoint) => {
+    // Generate all HTML style tags with a CDN prefix
+    const styles = chunksSorted.styles.map(chunkCss =>
+      `<link rel="stylesheet" href="https://cdn.domain.com/${chunkCss}" />`
+    ).join('');
+
+    // Generate all HTML style tags with CDN prefix and defer attribute
+    const scripts = chunksSorted.scripts.map(chunkJs =>
+      `<script defer src="https://cdn.domain.com/${chunkJs}"></script>`
+    ).join('');
+
+    return { styles, scripts };
+  }
+});
+```
+
+T> The arrow function syntax allow you to access the class properties.
+
+W> The function provides more flexibility by replacing the default behavior. Follow the example above to make sure it works.
+
+The function must return an object with the following format:
+
+```json
+{
+  "styles": "",
+  "scripts": ""
+}
+```
+
+The `customFormatTags` function has two parameters:
+
+#### `chunksSorted`
+
+`object`
+
+The list of the chunks sorted by type: `styles` and `scripts`.
+
+#### `Entrypoint`
+
+`Object`
+
+The object is included in every single ChunkGroup. The variable contains all information about the current entry point; log it on the console for more details.
+
+W> Use this variable only for a full customization if the `chunksSorted` variable does not meet your needs.
+
+### `generateChunksManifest`
+
+`boolean = false`
+
+Tells the plugin whether to generate the `chunks-manifest.json`. The file contains the list of all the chunks grouped by entry points.
+
+```js
+new ChunksWebpackPlugin({
+  generateChunksManifest: false
```

Comment:
It is common to showcase a non-default setting
```suggestion
  generateChunksManifest: true
```

Reply:
Thanks for the advice!